lib/index.js

```diff
@@ -95,6 +99,17 @@ function logFacesAppender(config) {
         // no event object returned, consider the event ignored
         return;
       }
+      if (
+        typeof lfsEvent !== 'object' ||
+        Array.isArray(lfsEvent) ||
+        Object.keys(lfsEvent).length < 3
```

Comment:
Can I know the rationale of why `< 3`?

Reply:
There were 3 props I specified are required in the type: `[t,p,a]` which would likely be problematic if sent without, but I didn't want to do explicit checks for those, but we could do this instead, with more explicit checks:

```suggestion
        !lfsEvent.t || lfsEvent.p == null || lfsEvent.a == null
```

we could also simplify and just say

```suggestion
        !Object.keys(lfsEvent).length
```

Reply:
Was thinking of maybe `< 5` since minimally `a` `t` `p` `g` `m` is always present.
What do you think?

```suggestion
        Object.keys(lfsEvent).length < 5
```

Reply:
4 because `m` will be allowed to be ignored for this usecase in particular.

Reply:
Pardon my memory, `4` it is. 😂
Mind helping to add the comment for the rationale of 4 somewhere?

```suggestion
        Object.keys(lfsEvent).length < 4
```

Reply:
I'll do you one better! A constant variable with a very obvious name and a comment 🚀